sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -323,6 +323,38 @@ To see how to use the Document Translation client library with Azure Storage Blo
 for your containers, and download the finished translated documents, see this [sample][sample_translation_with_azure_blob].
 Note that you will need to install the [azure-storage-blob][azure_storage_blob] library to run this sample.
 
+## Advanced Topics
+
+The following section provides some insights for some of the advanced translation features such as glossaries and custom translation models.
+
+### **Glossaries**
+Glossaries are domain-specific dictionaries. For example, if you want to translate some medical-related documents, you may need support for the many words, terminology, and idioms in the medical field which you can't find in the standard translation dictionary or you simply need specific translation. This is why Document Translation provides support for glossaries. 
+
+#### **How To Create Glossary File**
+
+Document Translation supports glossaries in the following formats:
+
+|**File Type**|**Extension**|**Description**|**Samples**|
+|---------------|---------------|---------------|---------------|
+|Tab-Separated Values/TAB|.tsv, .tab|Read more on [wikipedia][tsv_files_wikipedia]|[glossary_sample.tsv][sample_tsv_file]|
+|Comma-Seperated Values|.csv|Read more on [wikipedia][csv_files_wikipedia]|[glossary_sample.csv][sample_csv_file]|
+|Localization Interchange File Format|.xlf, .xliff|Read more on [wikipedia][xlf_files_wikipedia]|[glossary_sample.xlf][sample_xlf_file]|
```

Comment:
let's just link to this table in the documentation since it's inevitable it will change as they add more formats: https://docs.microsoft.com/azure/cognitive-services/translator/document-translation/overview#supported-glossary-formats

I think we should still call out that there are glossary samples in the samples folder, though.